From alan@lxorguk.ukuu.org.uk  Wed Aug 12 13:15:51 2009
From: Alan Cox <alan@lxorguk.ukuu.org.uk>
Date: Fri, 07 Aug 2009 19:23:11 +0100
Subject: Staging: sep: kill lock wrapper
To: greg@kroah.com, mark.a.allyn@intel.com
Message-ID: <20090807182310.19360.65425.stgit@localhost.localdomain>


From: Alan Cox <alan@linux.intel.com>

We don't need wrappers for this so clean them up

Signed-off-by: Alan Cox <alan@linux.intel.com>
Signed-off-by: Greg Kroah-Hartman <gregkh@suse.de>
---

 drivers/staging/sep/sep_main_mod.c |   18 ------------------
 1 file changed, 18 deletions(-)


--- a/drivers/staging/sep/sep_main_mod.c
+++ b/drivers/staging/sep/sep_main_mod.c
@@ -288,24 +288,6 @@ static int sep_lock_user_pages(unsigned 
 -----------------------------------------------*/
 
 /*
-  this function locks SEP by locking the semaphore
-*/
-int sep_lock()
-{
-	mutex_lock(&sep_mutex);
-	return 0;
-}
-
-/*
-  this function unlocks SEP
-*/
-void sep_unlock()
-{
-	/* release mutex */
-	mutex_unlock(&sep_mutex);
-}
-
-/*
   this function returns the address of the message shared area
 */
 void sep_map_shared_area(unsigned long *mappedAddr_ptr)
